bazel/repository_locations.bzl

```diff
@@ -45,6 +45,22 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2019-10-10",
         use_category = ["build"],
     ),
+    rules_fuzzing = dict(
+        project_name = "Fuzzing Rules for Bazel",
+        project_desc = "Bazel rules for fuzz tests",
+        project_url = "https://github.com/bazelbuild/rules_fuzzing",
+        version = "2c97fcd74f20dce950c77129b9b9b31cc63a4c8e",
+        sha256 = "6764e198db6f84bd1f6bf9565ebd7bb532a73b5f99cfe19126a279f67cc9b633",
+        strip_prefix = "rules_fuzzing-{version}",
+        urls = ["https://github.com/bazelbuild/rules_fuzzing/archive/{version}.tar.gz"],
+        release_date = "2021-01-28",
+        use_category = ["test_only"],
+        implied_untracked_deps = [
+            # This is a repository rule generated to define an OSS-Fuzz fuzzing
+            # engine target from the CFLAGS/CXXFLAGS environment.
+            "rules_fuzzing_oss_fuzz",
+        ],
+    ),
```

Comment:
Not a great OSSF score but this is test only. Looks OK to me. @htuch?
```
scorecard --repo=https://github.com/bazelbuild/rules_fuzzing
RESULTS                    
-------                    
Active: Pass 10            
Branch-Protection: Fail 0  
CI-Tests: Pass 10          
CII-Best-Practices: Fail 10
Code-Review: Pass 10       
Contributors: Fail 10      
Frozen-Deps: Fail 5        
Fuzzing: Pass 10           
Packaging: Fail 10         
Pull-Requests: Pass 10     
SAST: Fail 10              
Security-Policy: Fail 10   
Signed-Releases: Fail 0    
Signed-Tags: Fail 0        
```

Reply:
TIL about Scorecard, this is awesome! :) Just as a heads-up, we are planning to improve upon a number of these items in the short term, and will update the Envoy dependencies when this happens.